### ISSUE TITLE
I using Pdf-node-creator for generating pdf in my node application, It was working fine in local environment , But in production I get the below Error

### ISSUE THREAD

**Issue by ghost** [marcbachmann/node-html-pdf#665]
[Error: ENOENT: no such file or directory, open 'C:\tmp\html-pdf-11516.pdf'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\tmp\\html-pdf-11516.pdf'
}

**Comment by skulas** [comment]
is your production environment also windows?
If so, maybe permissions issues on c:\tmp.
If this is a docker image, you need to make sure you have write access and a volume attached to it.

**Comment by ghost** [comment]
yes my production environment is also windows, no there is no temp file in c:/ .
Actually I found the answer that in virtual machines we don't have permission to use temp memory so we got this error.

**Comment by scush989898** [comment]
i got the same problem on production using heroku. The solution for me was creating a new env var OPENSSL_CONF with value /dev/null.

https://prnt.sc/VaOabzLAXFlO

i added to heroku env vars and it worked for me.

**Comment by panshak** [comment]
@scush989898 Thank you very much. I struggled with this problem for close to 5 days now. Your solution worked for me

**Comment by panshak** [comment]
@scush989898 please will you mind to share why this solution fixed the problem?
